lib/cloud_controller/permissions.rb

```diff
@@ -107,6 +107,10 @@ def readable_org_guids_query
     membership.authorized_org_guids_subquery(ROLES_FOR_ORG_READING)
   end
 
+  def can_delete_buildpack_cache?(space_id)
+    roles.admin? || membership.role_applies?(ROLES_FOR_SPACE_SERVICES_READING, space_id)
```

Comment:
Clearing the cache isn't a "READ" action. I think if we want `SPACE_DEVELOPER` and `SPACE_SUPPORTER` to be able to do this then the `ROLES_FOR_APP_MANAGING` constant is a bit more correct.

Reply:
That is indeed true